src/css/stake-page.scss

```diff
@@ -53,6 +53,61 @@
     }
   }
 
+  .stake-options {
```

Comment:
Can we use `mobile-first` appoarch? I mean, we should start from the smallest dimensions to the largest.